Gemfile

```diff
@@ -26,6 +27,8 @@ gem 'paranoia'
 gem 'pg'
 gem 'puma'
 gem 'pundit'
+gem 'rails-i18n', '~> 5.0.0'
+gem 'ransack', github: 'activerecord-hackery/ransack'
```

Comment:
Do we need the github version of the gem? I whould prefer the rubygems version for stability reasons